contents/handbook/small-teams/experimentation/objectives.mdx

```diff
@@ -10,6 +10,29 @@
 * **Rationale:**
  * Make sure that experimentation is a first class feature, Posthog users are excited to use alongside the rest of the analytics features
 
+
+----- Suggested replacement for Experimentation focus
+
+Objective: Ensure that feature flags can mature with users
+
+* **Key Result:** X users on permission and alerting features
+    - alerting/notification hooks to slack
+    - access control
+* **Key Result:** Qualitatively improve interface
+    - highlight what’s completely rolled out
+    - “last updated” column on table
+    - (cohort creation improvement, flexible user additions)
+* **Key Result:** Increase optional metric impact analysis by X%
```

Comment:
I'm not convinced this should be here, since not all feature flags require metrics, and we already have a way to create insights out of feature flags.

The other 3 look great to me though!

Reply:
(I don't intuitively get this one either)

Reply:
Yeah, I am a bit unsure here either. "make metric analysis selection part of the feature flag creation flow" <- isn't that what experiments are supposed to do?

Reply:
Idea here is to have metric impact analysis as a suggested option when creating the feature flag. Experiments is useful for deciding between variants how much a metric can be improved or not. 

Feature flag metric tracking is similar but the default behavior we should be encouraging is to always be testing/monitoring against the control. Feature flag rollouts should always be comparing impact which can be done with highlighting possible spikes in error, sudden drops in usage, or changes in conversion. 

Context: I think we have a narrow conception of feature flags where it's just a simple switch to turn on and off, but feature flags themselves are basic experiments (a variant vs a control). The intention of rolling out to beta users, internal users, and/or a subset of users is so we can evaluate metric changes and bugs that are happening before doing a wider rollout but currently we don't really have good support for that—it's a major afterthought. By improving usage of a basic form of experimentation, my guess is that we'll have an easier time transitioning folks towards experimentation where you're testing multiple variations.

Reply:
Agree that we should encourage tracking where it makes sense. Disagree on it being a part of creation flow, as that's yet another step we're adding that's not useful to everyone.

In my eyes, this is part of the KR above, i.e. qualitatively improving the interface. 

When viewing a feature flag, we already have this part now:

<img width="609" alt="image" src="https://user-images.githubusercontent.com/7115141/195370105-b04f6fd1-5110-46cd-81a3-7d566300ebf1.png">


and I think that "create Insight" button already does what this KR is about. Maybe we can play around with the wording a bit / somehow promote using feature flag insights; but that doesn't _seem_ to me to be like a standalone key result, specially for feature flags.